ci/scripts/gitlab/artifactory_upload.sh

```
@@ -83,47 +64,8 @@ fi
-                echo "Uploading ${WHEEL_FILE} to ${ARTIFACTORY_PATH}..."
-
-                CI=true jf rt u --fail-no-op --url="${NAT_ARTIFACTORY_URL}" \
-                    --user="${URM_USER}" --password="${URM_API_KEY}" \
-                    --flat=false "${WHEEL_FILE}" "${ARTIFACTORY_PATH}" \
-                    --target-props "arch=${NAT_ARCH};os=${NAT_OS};branch=${GIT_TAG};component_name=${ARTIFACTORY_COMPONENT_FIXED_NAME};version=${GIT_TAG};release_approver=${RELEASE_APPROVER};release_status=${RELEASE_STATUS}"
-            done
-        done
-    done
-    echo "All wheels uploaded to Artifactory."
+    echo "Uploading wheels to Artifactory (${NAT_ARTIFACTORY_NAME}) for ${GIT_TAG}..."
+    python ${GITLAB_SCRIPT_DIR}/artifactory_upload.py
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)